packages/frontend/src/components/TableTree.tsx

```diff
@@ -175,6 +175,7 @@ const NodeItemButtons: FC<{
                     dimension.label
                 } on the table ${dimension.tableLabel}`,
                 type,
+                ...(type === MetricType.AVERAGE ? { round: 2 } : {}),
```

Comment:
hardcoded round for the table, :+1:

Reply:
Yep as per the ticket but your suggestion is actually much better